sample/breakpoints.css.scss

```diff
@@ -0,0 +1,85 @@
+
+@import '../grid-system.scss';
+
+.app {
+	@include vui-gs-container;
+}
+
+.app * {
+	box-sizing: border-box;
+}
+
+.header {
+	@include vui-gs-row;
+	background-color: rgba(0,0,250,0.4);
+	height: 60px;
+	text-align: center;
+}
+
+.stuff-1 > div {
+	@include vui-gs-span(12 break);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+#bp-extra-small {
+	display: block;
+}
+
+#bp-small {
+	display: none;
+}
+
+#bp-medium {
+	display: none;
+}
+
+@include vui-gs-small() {
+
+	#bp-extra-small {
+		display: none;
+	}
+
+	#bp-small {
+		display: block;
+	}
+
+	#bp-medium {
+		display: none;
+	}
+
+	.stuff-1 > div {
+		@include vui-gs-span(4 nobreak);
+	}
+
+	.stuff-1 > div:nth-child(3),
+	.stuff-1 > div:nth-child(6) {
+		@include vui-gs-span(last 4);
+	}
+
+}
+
+@include vui-gs-medium() {
+
+	#bp-extra-small {
+		display: none;
+	}
+
+	#bp-small {
+		display: none;
+	}
+
+	#bp-medium {
+		display: block;
+	}
+
+	.stuff-1 > div,
+	.stuff-1 > div:nth-child(3) {
+		@include vui-gs-span(2 nobreak);
+	}
+
+	.stuff-1 > div:nth-child(6) {
```

Comment:
The selectivity in this case is tricky.  As mentioned previously, because there is no max-width constrain in the media query, the consumer must be careful to properly override the styles set in queries for smaller container sizes.  In this case, `.stuff-1 > div:last-child` would have worked if I hadn't used `nth-child` in the `small` mixin above.  This is an implementation detail for this particular page setup, but as you can see, there is potential with the media queries as defined for some frustration if people don't understand what's happening.  @dlockhart - be interest in your opinion regarding how the queries are defined (see my other comment specific to the breakpoint mixins.